src/main/java/org/assertj/core/api/AbstractIterableAssert.java

```diff
@@ -2464,6 +2464,15 @@ public SELF anySatisfy(Consumer<? super ELEMENT> requirements) {
     return myself;
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public SELF noneSatisfy(Consumer<? super ELEMENT> requirements) {
+    iterables.assertAnySatisfy(info, actual, requirements);
```

Comment:
should be `assertNoneSatisfy`

Reply:
Add a test like for `https://github.com/joel-costigliola/assertj-core/blob/cc178abc5871bc34525113550373ffd90e303034/src/test/java/org/assertj/core/api/iterable/IterableAssert_allSatisfy_Test.java` but for `noneSatisfy` for each assert type exposing  `noneSatisfy` (that would have caught the previous mistake(.